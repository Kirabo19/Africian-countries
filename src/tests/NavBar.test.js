import React from 'react';
import TestRenderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/Store';
import Navbar from '../component/Navbar';

it('Navbar Component snapshot test', () => {
  const NavbarRender = TestRenderer
    .create(
      <Provider store={store}>
        <Navbar />
      </Provider>,
    )
    .toJSON();
  expect(NavbarRender).toMatchSnapshot();
});
