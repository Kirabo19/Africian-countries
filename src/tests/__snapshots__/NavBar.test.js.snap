// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Component snapshot test 1`] = `
<nav
  className="navbar df-nav"
>
  <div
    className="container"
  >
    <h1>
      African Countries
    </h1>
    <div
      role="button"
      tabIndex={0}
    >
      Change Theme
    </div>
  </div>
</nav>
`;
